showcase/app/templates/page-components/code-editor.hbs

```diff
@@ -172,4 +172,22 @@
       />
     </SF.Item>
   </Shw::Flex>
+
+  <Shw::Flex @direction="column" as |SF|>
+    <SF.Item @label="Update value externally">
+      <Hds::Button
+        @text="Change value"
+        class="hds-code-editor-controls"
+        {{on "click" (fn this.setExternalUpdateDemo "New value")}}
```

Comment:
So this click handler and the `onInput` function are the same (they both point to `this.setExternalUpdateDemo`). This causes an issue though when you click the button that I think goes like this:

1. button click sets `externalUpdateDemo` value
2. that passes to the modifier as the `value` named argument
3. this triggers a call into CodeMirror to set its value (which was added to in this PR 🎉 )
4. this triggers the `EditorView.updateListener` call
5. which calls the `onInput` function, also passed to the modifier as a named arg
6. which... sets `externalUpdateDemo` again

In this case if you wanted to use the same function for `onInput` and external setting of a value passed to `value`, then that function should be debounced, not doing the debounce in the modifier itself.